### ISSUE TITLE
Location appears twice on search facets

### ISSUE THREAD

**Issue by vantuyls** [OregonDigital/OD2#537]
### Descriptive summary
Location facet appears twice on the main search interface. One of these facets shows location labels, the other shows URIs.

### Expected behavior
Location facet appears once

### Related work

#271

**Comment by CGillen** [comment]
QA:
- [ ] Location label only appears once in search facets
- [ ] Water Basin is a controlled vocab
- [ ] Ranger District is a controlled vocab

**Comment by raybrarian** [comment]
QA failed: not seeing Location facet (or location label in item page):
![image](https://user-images.githubusercontent.com/4460510/58716170-47302900-837d-11e9-96e7-94d3f3cd760d.png)

**Comment by CGillen** [comment]
Separate issue (#552) is resolved. Try again

**Comment by vantuyls** [comment]
QA pass. Location appears only once and facets show labels.
